dist/crawl_links.js

```diff
@@ -0,0 +1,45 @@
+/*
+Collects same-origin in-viewport links from img, h[1-6] tags,
+sorted by size from largest to smallest.
+This is the seed for generating a list of URLs to crawl from the given page.
```

Comment:
Do we also need any logic in the custom metric itself to return an empty list if the given page is already a secondary test page? Or will that be handled by the test server?